kitsune/search/base.py

```diff
@@ -95,17 +95,30 @@ def migrate_reads(cls):
     def _update_alias(cls, alias, new_index):
         client = es_client()
         old_index = cls.alias_points_at(alias)
-        if not old_index:
-            client.indices.put_alias(new_index, alias)
-        else:
-            client.indices.update_aliases(
-                {
-                    "actions": [
-                        {"remove": {"index": old_index, "alias": alias}},
-                        {"add": {"index": new_index, "alias": alias}},
-                    ]
-                }
-            )
+
+        # Handle special case where an index with same name as alias exists
+        try:
```

Comment:
Is there a reason for adding this check?

Reply:
I think I can drop this; I added this when troubleshooting test issues with index and alias collisions. I'll see if dropping it causes problems to recur.